cuda/apps/CMakeLists.txt

```diff
@@ -22,6 +22,8 @@ if(NOT build)
 endif()
 
 if(VTK_FOUND AND WITH_OPENNI)
-  PCL_CUDA_ADD_EXECUTABLE(kinect_viewer_cuda COMPONENT ${SUBSYS_NAME} SOURCES src/kinect_viewer_cuda.cpp)
+  PCL_ADD_EXECUTABLE(kinect_viewer_cuda COMPONENT ${SUBSYS_NAME} SOURCES src/kinect_viewer_cuda.cpp)
```

Comment:
How about keeping the function, and replacing it's innards with an indirection to `PCL_ADD_EXECUTABLE` and `target_compile_options` call?

Reply:
Yeah, we could go with that.